packages/venia-ui/lib/components/Main/main.css

```diff
@@ -12,7 +12,8 @@
 .page {
     margin: 0 auto;
     max-width: var(--venia-global-maxWidth);
-    min-height: 100vh;
+    min-height: 40rem;
+    padding: 1rem 0 1.5rem;
```

Comment:
I don't think this will work - some components are meant to fit snug against the header ie error page and the home page sample content. Notice the gap:

![](https://i.gyazo.com/7fc95d5078901f53e42cc88257c1261b.jpg)